tests/test_checkout.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import os
+import sys
+import unittest
+
+import requests
+
+from dss.config import override_bucket_config, BucketConfig
```

Comment:
historically, we need to do the dss imports _after_ the path hack in order to make the tests runnable from the command line.  I don't know if anyone still runs it that way though.  @kislyuk might have been the last holdout...?

Reply:
I like to be able to run the tests from the command line, yes :)

Please do move the path hack above own package (dss) imports. Otherwise there is no reason to use the path hack at all.